python/ray/monitor.py

```diff
@@ -75,6 +76,9 @@ def __init__(self, redis_address, autoscaling_config, redis_password=None):
             redis_address, redis_password=redis_password)
         self.redis = ray._private.services.create_redis_client(
             redis_address, password=redis_password)
+        self.global_state_accessor = GlobalStateAccessor(
+            redis_address, redis_password, False)
+        self.global_state_accessor.connect()
```

Comment:
Why don't we run `_initialize_global_state`?

Reply:
`_initialize_global_state` is a function in `state.py` and it init a global state accessor inside but not explode it.

What we need here is init a global state accessor and use it to get heartbeats from gcs, which is not included in `state.py`